test/bench/avro-upsert/mzcompose.yml

```diff
@@ -166,11 +166,12 @@ mzworkflows:
     - step: workflow
       workflow: start-services
     - step: run
-      service: avro-upsert-generator
+      service: kafka-generator
```

Comment:
Maybe let's call this `kafka-avro-generator`?

Reply:
👍